File: 2st/Yeonwoo/각도기.java
class Solution {
    public int solution(int angle) {
        int answer = 0;
        if(angle>0 && angle <90) {
            return answer = 1;
        }
        else if(angle==90) {
            return answer = 2;
        }
        else if(angle>90 && angle <180) {
            return answer = 3;
        }
        else if(angle==180) {
            return answer = 4;
        }
        return answer;
    }
}

File: 4st/Eunseong/각도기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int angle) {
    int answer = 0;
    if (angle < 90) {
        answer = 1;
    }
    if (angle == 90) {
        answer = 2;
    }
    if (90 < angle && angle < 180) {
        answer = 3;
    }
    if (angle == 180) {
        answer = 4;
    }
    return answer;
}